bug/bug.go

```diff
@@ -681,3 +683,65 @@ func (bug *Bug) Compile() Snapshot {
 
 	return snap
 }
+
+// Participants return the participants list of the bug.
+// Participants are identities that added comments to a
+// bug or created it.
+func (bug *Bug) computeParticipants() []identity.Interface {
+	participants := make([]identity.Interface, 0, 1)
+
+	it := NewOperationIterator(bug)
```

Comment:
Should i loop only over packed operations instead of using `NewOperationIterator`?